lib/dest/write-contents/write-symbolic-link.js

```diff
@@ -18,25 +25,41 @@ function writeSymbolicLink(file, optResolver, onWritten) {
   //    For example, JetBrains product lines will delete the entire
   //    contents of the TARGET directory because the product does not
   //    realize it's a symlink as the JVM and Node return false for isSymlink.
-  var useJunctions = optResolver.resolve('useJunctions', file);
+  var useJunctions = isWindows && optResolver.resolve('useJunctions', file);
 
-  var symDirType =  useJunctions ? 'junction' : 'dir';
-  var symType = isDirectory ? symDirType : 'file';
   var isRelative = optResolver.resolve('relativeSymlinks', file);
+  var flag = optResolver.resolve('flag', file);
 
-  // This is done after prepare() to use the adjusted file.base property
-  if (isRelative && symType !== 'junction') {
-    file.symlink = path.relative(file.base, file.symlink);
+  if (isWindows) {
+    fs.stat(file.symlink, onStat);
+  } else {
+    onStat(null, file.stat);
   }
 
-  var flag = optResolver.resolve('flag', file);
+  function onStat(statErr, stat) {
+    if (statErr && statErr.code !== 'ENOENT') {
```

Comment:
Should we use a `isFatal*` method for this?

Reply:
Maybe, sure.